LeaderboardBackend.Test/Categories.cs

```diff
@@ -4,6 +4,7 @@
 using System.Net.Http.Json;
 using System.Threading.Tasks;
 using FluentAssertions.Specialized;
+using LeaderboardBackend.Migrations;
```

Comment:
I don't think this is supposed to be here.

Reply:
Dunno why `dotnet format` doesn't catch that for me.